BetaUtils.Core/BetaUtils.Core.csproj

```diff
@@ -2,12 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>10</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Extensions\" />
     <Folder Include="FeatureFlags\" />
     <Folder Include="Validation\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Hellang.Middleware.ProblemDetails" Version="3.1.0" />
```

Comment:
What are we using this library for? Is this something we can do ourselves? If so, maybe we should consider doing it, so we can avoid external dependencies

Reply:
It is not something we can do our self, it handles exceptions same as 
`
<PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.2.0" /> 
`
`
<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
`
Ref: https://github.com/khellang/Middleware

but since using Microsoft dependencies would be more acceptable by clients, I will change to Mvc.Core and Abastractions.